examples/istio-multi-cluster/0.certs-tool/README.md

```diff
@@ -0,0 +1,26 @@
+# Generating Certificates for Bootstrapping Multicluster / Mesh Expansion Chain of Trust
+
+The directory contains two Makefiles for generating new root, intermediate certificates and workload certificates:
+- `Makefile.k8s.mk`: Creates certificates based on a root-ca from a k8s cluster. The current context in the default
+`kubeconfig` is used for accessing the cluster.
+- `Makefile.selfsigned.mk`: Creates certificates based on a generated self-signed root.
```

Comment:
The directory contains two Makefiles for generating new root, intermediate and workload certificates:
- `Makefile.k8s.mk`: Creates certificates based on a root-ca from a Kubernetes cluster. The current context in the default `kubeconfig` is used to access the cluster.
- `Makefile.selfsigned.mk`: Creates certificates based on a generated self-signed root.